### PR TITLE
openark: Add version 1.1.0

### PR DESCRIPTION
**Pull request by wenmin92**
[OpenArk](https://openark.blackint3.com/) is an open source anti-rookit(ARK) tool for Windows. Ark is Anti-Rootkit abbreviated, it aimmed at reversing/programming helper and also users can find out hidden malwares in the OS.

### CODE REVIEW

File: bucket/openark.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "64bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk64.exe",
+            "hash": "6fad8b0553d0ab5464f2547b8890696b0bcf4e1d8ea89b29e6c9840389b3a95e"
+        },
+        "32bit": {
+            "url": "https://github.com/BlackINT3/OpenArk/releases/download/v1.1.0/OpenArk32.exe",
+            "hash": "550056c99590a902d9df4a2dfbf75da9f045dd8fb25099b30b34f0ca3937a2e1"
+        }
+    },
+    "pre_install": "Rename-Item \"$dir\\OpenArk*.exe\" \"OpenArk.exe\"",
```
Comment (Ash258) [inline bucket/openark.json:16]: Rewrite URL

```suggestion
```

File: bucket/openark.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "bin": "OpenArk.exe",
```
Comment (Ash258) [inline bucket/openark.json:17]: Available command line arguments?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/openark.json (hunk 2 above)
Comment (wenmin92) [reply bucket/openark.json:17]: No, just we can open UI from command line.

**Review by Ash258 — CHANGES_REQUESTED** [review]
https://github.com/lukesampson/scoop-extras/pull/5869#discussion_r597384535

**Comment by wenmin92** [comment]
> [#5869 (comment)](https://github.com/lukesampson/scoop-extras/pull/5869#discussion_r597384535)

Oh, sorry, I've rename the url, but forgot to change the autoupdate, and the autoupdate rewrite the url.

File: bucket/openark.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "checkver": {
+        "github": "https://github.com/BlackINT3/OpenArk"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/BlackINT3/OpenArk/releases/download/v$version/OpenArk64.exe#/openark.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/BlackINT3/OpenArk/releases/download/v$version/OpenArk32.exe#/openark.exe"
```
Comment (Ash258) [inline bucket/openark.json:31]: It is not `openark.exe` 

image

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/openark.json (hunk 3 above)
Comment (wenmin92) [reply bucket/openark.json:31]: Actually, the windows will ignore the case. 😂 But I will change to it's proper case.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**